### ISSUE TITLE
Testing the seed dependency and hyperparameter tunning for the linear case

### ISSUE THREAD

**Issue by Mekahou** [HighDimensionalEconLab/symmetry_dynamic_programming#3]
- [x] Change the learning rate so it converges in 10ish epochs
- [x] test it for different seeds and see it its convergence (1,12,123,1234,42,43, 10, 11)
